android/modules/ui/src/java/ti/modules/titanium/ui/widget/webview/TiUIWebView.java

```diff
@@ -599,13 +600,17 @@ public void setUrl(String url)
 	{
 		reloadMethod = reloadTypes.URL;
 		reloadData = url;
-		String finalUrl = url;
-		Uri uri = Uri.parse(finalUrl);
-		boolean originalUrlHasScheme = (uri.getScheme() != null);
+		final Uri uri = Uri.parse(url);
 
-		if (!originalUrlHasScheme) {
-			finalUrl = getProxy().resolveUrl(null, finalUrl);
-		}
+		// Extract URL query parameters.
+		final String query = uri.getQuery() != null ? "?" + uri.getQuery() : null;
```

Comment:
We shouldn't set `query` variable to `null` because appending it to `finalUrl` down below will create an invalid URL. It should be an empty string.

Also it looks like this change might break anchor tag support. For example:
https://en.wikipedia.org/wiki/Appcelerator#Products

Reply:
Good catch, updated PR